tremr/ViewControllers/ExerciseViewController.swift

```diff
@@ -8,8 +8,11 @@
 
 import UIKit
 
-class ExerciseViewController: UIViewController {
-
+class ExerciseViewController: UIViewController {    
+    @IBAction func mainViewTransition(_ sender: Any) {
```

Comment:
Does this add an animation when the exercise view loads? Or is this the handler for the back button?

Reply:
This tells the view controller that there is a button, it's a handler. 

This looks like the back button in my video in the middle of the page, but it's now placed in the upper left.